scripts/ock-once-swtok.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+PIN=${OCK_PIN:-"12345678"}
+
+if id -u | grep -qw 0 && groups | grep -qwv pkcs11 ; then
```

Comment:
what's this?

Reply:
This is a check for the basic requirements of pkcsslotd. I put it in to fail early in this script. It will be removed/modified, as soon as the requirements for group membership or being root has been removed.